scripts/data_generator.py

```
@@ -0,0 +1,141 @@
+    logging.info("Finished synthetic data generation.")
+    return synthetic_data
+
+if __name__ == '__main__':
+    parser = argparse.ArgumentParser()
+    parser.add_argument("--horizon", default="1M", help="Market horizon to generate data for")
+    args = parser.parse_args()
+
+    try:
+        data = generate_synthetic_data(args.horizon)
+        print(json.dumps(data))
+    except Exception as e:
```

Ruff (0.14.1)

Do not catch blind exception: `Exception`

(BLE001)